### PR TITLE
feat(Pipeline Migration to 1ES) - User Story 2126528(Accessibility Insights Web SDT - CI)

### PR DESCRIPTION
**Pull request by v-viyada**
#### Details

Migrated Pipeline Accessibility Insights Web SDT - CI to 1ES with the help of migration tool.
Migrated Pipeline is tested successfully and checked the artifacts. It looks good.

##### Motivation

Associated User Story : 2126528
migration of pipeline https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=49447&view=results

##### Context

Raised Support post with 1ES team on how to use dynamic tsa.config in source sdl stage. We will raise separate PR, if any better approach or solution is provided.

#### Pull request checklist

- [na] Addresses an existing issue: #0000
- [x] Ran `yarn fastpass`
- [na] Added/updated relevant unit test(s) (and ran `yarn test`)
- [na] Verified code coverage for the changes made. Check coverage report at: `<rootDir>/test-results/unit/coverage`
- [x] PR title *AND* final merge commit title both start with a semantic tag (`fix:`, `chore:`, `feat(feature-name):`, `refactor:`). See `CONTRIBUTING.md`.
- [ ] (UI changes only) Added screenshots/GIFs to description above
- [ ] (UI changes only) Verified usability with NVDA/JAWS

### CODE REVIEW

File: pipeline/create-tsa-options.ps1 [hunk 1]
```diff
@@ -0,0 +1,94 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (DaveTryon) [inline pipeline/create-tsa-options.ps1:1]: Can we move this file to under the tools folder? We'll also need to update the path in the yaml file

Comment (v-viyada) [reply pipeline/create-tsa-options.ps1]: moved to tools folder

**Comment by DaveTryon** [comment]
@v-viyada, the build itself looks good. I noticed 3 things with https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=49939&view=results that we probably need to consider:
1. The build triggered automatically on a PR, but this pipeline should only trigger after committing to main. That makes me think that trigger may be slightly off.
2. There's still a warning during the `Validate Hosted Pool Information (1ES PT)` [step](https://dev.azure.com/accessibility-insights-private/Accessibility%20Insights%20(private)/_build/results?buildId=49939&view=logs&j=f0b91480-5af9-5bb8-88ea-d786c63830c6&t=3415d08c-7efb-5757-a12b-9ada478ac1fa&l=193). Something about the default branch?
3. When I manually queued a build in this pipeline (before I noticed that the PR had already triggered the build), ADO displays an error: `Encountered error(s): Could not find valid pipeline YAML file for this branch/tag`

These may all be symptoms of the same underlying issue...

**Comment by v-viyada** [comment]
@DaveTryon 

1. I have disabled the PR trigger in pipeline setting. It will not trigger on PR anymore.
2. This is know issue for GitHub base pipelines. Please refer [Bug 2119901](https://dev.azure.com/mseng/1ES/_workitems/edit/2119901): Address issue "Failed to fetch default branch"
3. This file is available only in users/v-viyada/SDTpipelineto1ES branch. so it will show this error for other branches. Once this PR is merged, the file will be available in main branch. Other branches have to take latest from main once this PR is merged.

**Review by DaveTryon — APPROVED** [review]

**Pull request merged**